test/unit/specs/components/Personalization/createComponent.spec.js

```diff
@@ -163,49 +125,35 @@ describe("Personalization", () => {
     });
 
     expect(isAuthoringModeEnabled).toHaveBeenCalled();
-    expect(getDecisionScopes).toHaveBeenCalled();
-    expect(hasScopes).toHaveBeenCalled();
-    expect(createQueryDetails).toHaveBeenCalled();
-    expect(mergeQuery).toHaveBeenCalledWith(event, eventQueryDetails);
-    expect(hideContainers).toHaveBeenCalled();
-
-    expect(onResponseHandler).not.toHaveBeenCalled();
+    expect(pageLoadHandler).not.toHaveBeenCalled();
+    expect(viewChangeHandler).toHaveBeenCalled();
+    expect(mergeQuery).not.toHaveBeenCalled();
     expect(onClickHandler).not.toHaveBeenCalled();
-    expect(showContainers).not.toHaveBeenCalled();
   });
-
-  it("should merge the query data, hide containers and trigger onResponseHandler", () => {
-    const renderDecisions = true;
-    const decisionScopes = ["foo"];
-    const eventQueryDetails = {
-      accepts: allSchemas,
-      decisionScopes
-    };
+  /*
+  it("should trigger pageLoadHandler if cache is initialized, viewName is not provided, decisionScopes is empty(not spa impl)", () => {
     build();
-    hasScopes.and.returnValue(true);
-    getDecisionScopes.and.returnValue("foo");
-    createQueryDetails.and.returnValue(eventQueryDetails);
-
-    const onResponse = func => {
-      func({});
-    };
+    const renderDecisions = true;
+    const decisionScopes = [];
+    viewCache.isInitialized.and.returnValue(true);
+    event.toJSON.and.returnValue({});
 
     personalizationComponent.lifecycle.onBeforeEvent({
       event,
       renderDecisions,
-      decisionScopes,
-      onResponse
+      decisionScopes
     });
 
     expect(isAuthoringModeEnabled).toHaveBeenCalled();
-    expect(getDecisionScopes).toHaveBeenCalled();
-    expect(hasScopes).toHaveBeenCalled();
-    expect(createQueryDetails).toHaveBeenCalled();
-    expect(mergeQuery).toHaveBeenCalledWith(event, eventQueryDetails);
-    expect(hideContainers).toHaveBeenCalled();
-    expect(onResponseHandler).toHaveBeenCalled();
-
+    expect(pageLoadHandler).toHaveBeenCalled();
+    expect(viewChangeHandler).not.toHaveBeenCalled();
+    expect(mergeQuery).not.toHaveBeenCalled();
     expect(onClickHandler).not.toHaveBeenCalled();
-    expect(showContainers).not.toHaveBeenCalled();
+  }); */
```

Comment:
Can we get rid of this or make it a useful test?

Reply:
yes, thank you, somehow I forgot it